pages/sdk-protocol-kit/reference.md

```diff
@@ -225,7 +225,7 @@ Returns a new instance of the Protocol Kit connected to a new Safe or a new Sign
 Connection of a deployed Safe using the `safeAddress` property:
 
 ```typescript
-const safeSdk = await safeSdk.connect({ ethAdapter, safeAddress })
+const protocolKit = await protocolKit.connect({ ethAdapter, safeAddress })
```

Comment:
I think we have a mistake here. Should be:

```suggestion
const protocolKit = await Safe.connect({ ethAdapter, safeAddress })
```

Reply:
mmm nope, it's ok like that